gensrc/script/doris_builtins_functions.py

```diff
@@ -747,6 +747,25 @@
     [['array_pushfront'], 'ARRAY_VARCHAR',    ['ARRAY_VARCHAR', 'VARCHAR'], 'ALWAYS_NULLABLE'],
     [['array_pushfront'], 'ARRAY_STRING',     ['ARRAY_STRING', 'STRING'], 'ALWAYS_NULLABLE'],
 
+    [['array_pushback'], 'ARRAY_BOOLEAN',    ['ARRAY_BOOLEAN', 'BOOLEAN'], 'ALWAYS_NULLABLE'],
```

Comment:
maybe use ARRAY_T is better ? think of it ?

Reply:
But I think we should keep the same format with other array functions .